libpod/runtime_img.go

```diff
@@ -854,50 +834,52 @@ func (r *Runtime) UntagImage(image *storage.Image, tag string) (string, error) {
 
 // RemoveImage deletes an image from local storage
 // Images being used by running containers can only be removed if force=true
-func (r *Runtime) RemoveImage(image *storage.Image, force bool) (string, error) {
+func (r *Runtime) RemoveImage(image *image.Image, force bool) error {
```

Comment:
Thought: we're probably going to need to have some way to render an image invalid after it's removed, so people can't keep trying to do things to an image that is no longer local

Reply:
long term agree with both comments.